lib/RAST_SDK/AnnotationUtils.pm

```diff
@@ -107,30 +107,33 @@ sub _get_genome {
 ## get the genome gff_contents
 #
 sub _get_genome_gff_contents {
-    my ($self, $obj_ref) = @_;
+    my ($self, $obj) = @_;
 
-    return [] unless ($self->_validate_KB_objref($obj_ref));
+    my $chk = $self->_validate_KB_objref_name($obj);
+    return [] unless $chk->{check_passed};
 
     my $gff_filename = catfile($self->{genome_dir}, 'genome.gff');
     my ($gff_contents, $attr_delimiter) = ([], "=");
 
     # getting gff_contents from $obj_ref if it is of genome type
-    my $obj_info = $self->_fetch_object_info($obj_ref);
+    my $obj_info = $self->_fetch_object_info($obj, $chk);
+    return [] unless $obj_info;
 
     my $in_type = $obj_info->[2];
     my $is_genome = ($in_type =~ /KBaseGenomes.Genome/ ||
                      $in_type =~ /KBaseGenomeAnnotatioVns.GenomeAnnotation/);
```

Comment:
This is pre-existing code, I know, but is `KBaseGenomeAnnotatioVns` a valid type?

it could also be simplified to 
```perl
my $is_genome = ($in_type =~ /KBase(Genome(Annotation)?)s\.\1/);

Reply:
Sharp eyes!  I use Emacs on the server `dev1` and sometimes forgot to hit `esc` before I copy/paste.

Reply:
I meant Vim instead of Emacs.